packages-next/types/src/core.ts

```diff
@@ -42,7 +42,7 @@ export type GraphQLSchemaExtension = {
 // TODO: don't duplicate this between here and packages/keystone/ListTypes/list.js
 export function getGqlNames({
   listKey,
-  itemQueryName: _itemQueryName,
+  itemQueryName: _itemQueryName = listKey,
```

Comment:
This change does nothing? it's still required in the argument?
```suggestion
  itemQueryName: _itemQueryName,
```

Reply:
Ah yep, that was part of something else I was trying, will remove 👍